velox/docs/functions/spark/string.rst

```diff
@@ -141,6 +141,14 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
         SELECT overlay('Spark SQL', 'tructured', 2, 4); -- "Structured SQL"
         SELECT overlay('Spark SQL', '_', -6, 3); -- "_Sql"
 
+.. spark:function:: repeat(string, n) -> varchar
+
+    Returns the string which repeats ``string`` ``n`` times. 
+    ``n`` must be less than or equal to 10000.
```

Comment:
It would be nice to add a limit on the total size of the result string. Maybe, something like in lpad / rpad. Does Spark has a limit?

```
template <bool lpad, bool isAscii, typename TOutString, typename TInString>
FOLLY_ALWAYS_INLINE void pad(
    TOutString& output,
    const TInString& string,
    const int64_t size,
    const TInString& padString) {
  static constexpr size_t padMaxSize = 1024 * 1024; // 1MB
```

Reply:
> Does Spark has a limit?

I think in spark the limit is int32.max, I think it's a too big value to fuzzer test.
https://github.com/apache/spark/blob/a50b30d7a02bf45f1ddb8db0be6779b1441e1d4d/common/unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java#L870

Reply:
> I think it's a too big value to fuzzer test.

More importantly this is too large for production. How about we add 1 MB limit to the size of the result string?

Reply:
I'm ok for 1 MB limit, @rui-mo , is this ok to you?

Reply:
@zhli1142015 Yes, it makes sense to me.

Reply:
Updated, thanks.